custom/icds/messaging/indicators.py

```diff
@@ -115,8 +140,6 @@ class LSSubmissionPerformanceIndicator(LSIndicator):
     def __init__(self, domain, user):
         super(LSSubmissionPerformanceIndicator, self).__init__(domain, user)
 
-        child_locations = [l.location_id for l in self.user.sql_location.child_locations()]
-        self.aww_user_ids = user_ids_at_locations(child_locations)
```

Comment:
so the way i had imagined this working was that we'd call `get_user_ids_by_location` for each child location and then build a map of user id: location id which we could then use as an index to self.awc_locations to get the location name when we need to.

i realize that's using couch instead of es, and i think for something like this i'd rather rely on the primary data source for this info